src/components/ConfigEditor/ConnectionConfig.tsx

```diff
@@ -47,6 +59,18 @@ const ConnectionConfig: React.FC<ConnectionConfigProps> = ({
 
   return (
     <FieldSet label="Connection Details">
+      <InlineField label="Azure cloud" labelWidth={26} tooltip="Select an Azure Cloud.">
+        <Select
+          options={azureClouds}
+          value={azureClouds.find(v => v.value === jsonData.azureCloud)}
+          onChange={(change: SelectableValue<AzureCloudType>) =>
+            updateJsonData('azureCloud', change.value ? change.value : AzureCloudType.AzurePublic)
+          }
+          isClearable={false}
+          width={60}
```

Comment:
I guess we can add the `required` property here so it's not empty? (even though I think it cannot be empty with the default value)